.github/workflows/modules-tests.yml

```diff
@@ -0,0 +1,18 @@
+name: Runs the tests for library modules
+run-name: Executed by ${{ github.actor }}
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  modules-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+
+      - name: Install Avocado to run tests
+        run: pip3 install git+https://github.com/avocado-framework/avocado@7998c0a6d1bdd8ce60235cd4d2aa04afbebfe00e#egg=avocado_framework
+
+      - name: Run the ar module test
+        run: ./tests/test-module.py metadata/autils/archive/ar.yml
```

Comment:
Can't we make this generic enough so we don´t need to change this file for new libraries?

Reply:
That one line (alone), no.  The reason is that the current form of the `test-module.py` script reads one metadata file only.

And while it'd be trivial to make `test-module.py` support multiple metadata files, I'm still trying to come up with a good enough structure for the job/suites/results.  One possibility is one `Job` per metadata file, each with many suites to match the different supported platforms.  The results produced would then either be parsed into a nice user-consumable web page -- or maybe even generated as part of an actual result plugin.

Long story short, I guess I'm trying not to commit to a behavior/structure that will definitely change soon.

Reply:
I got the idea, thanks. I would love to see this generic enough to avoid editing the file. But fair enough, let's improve after.